wordpress-mobile/WordPress-iOS
# Append locale as query parameter to TOS URL

Fix #9824 

As mentioned in [this comment](https://github.com/wordpress-mobile/WordPress-iOS/issues/9824#issuecomment-487056318), this PR passes the locale identifier as a parameter in the TOS URL. 

To test:
- Check out the branch
- Run the app on a device (or simulator) with the system set to a language other than English.
- Navigate to Me > App Settings > About WordPress for iOS > tap Terms of Service

@etoledom Since you reviewed the original code that is updated now, would you mind taking a look?
cc / @aforcier
